src/GlobalStyle.js

```diff
@@ -0,0 +1,13 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  font-size: 112.5%;
```

Comment:
this percentage is really weird ;)
you better use rem- if I remember right 1 rem equals 16px which should be default font-size in general, of course you can set it higher ;)

Reply:
112.5% is 18px based on the default of most browsers (which is 16px). By setting it in px, she does not interfere with potential user settings. 

In modern web design, a bigger font size like this is often chosen, because of a chance in user consuming behavior. It's more "lay back" than it used to be. 

I would recommend the exact approach that is shown here.